tasks.py

```diff
@@ -718,7 +735,11 @@ def unittest(
         for individual_exclude_tag in exclude_tag:
             command += f" --tag {individual_exclude_tag}"
 
-    run_command(context, command)
+    env = {"NAUTOBOT_TEST_DB_NAME": test_db_name}
+    if xml_output:
+        env["NAUTOBOT_TEST_OUTPUT_DIR"] = _TEST_RESULTS_DOCKER_DIR
```

Comment:
Probably need alternative logic for `context.nautobot.local` test environment case, as `_TEST_RESULTS_DOCKER_DIR` won't exist on a local filesystem.